app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,113 @@
+class TasksController < ApplicationController
+  
+  
+  def index
+    @greeting = "HELLO, THERE!"
+    @tasks = Task.all
+  end
+  
+  
+  def show
+    task_id = params[:id].to_i
+    @task = Task.find_by(id: task_id)
+    if @task.nil?
+      flash[:error] = "Could not find task with id: " + task_id.to_s
```

Comment:
Soon we will learn flash and how it can show up on the app! Until then, you probably couldn't meaningfully use this (although I appreciate the test)